web/src/app/alerts/CreateAlertDialog/StepContent/CreateAlertServiceSelect.js

```diff
@@ -136,7 +132,7 @@ export function CreateAlertServiceSelect(props) {
         id={id}
         className={classes.serviceChip}
         onClick={(e) => e.preventDefault()}
-        onDelete={() => props.onChange(value.filter((v) => v !== id))}
+        onDelete={() => onChange(value.filter((v) => v !== id)[0])}
```

Comment:
Use `find(...)` instead of `filter(...)[0]`
- It won't crash if none are found
- It will stop processing the array at first find
- It won't allocate a new array as `filter` does

```suggestion
        onDelete={() => onChange(value.find((v) => v !== id))}
```